aws-sqs-http-connector/main.go

```diff
@@ -0,0 +1,190 @@
+package main
+
+import (
+	"errors"
+	"io/ioutil"
+	"log"
+
+	"net/http"
+	"os"
+
+	"go.uber.org/zap"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sqs"
+	"github.com/fission/keda-connectors/common"
+)
+
+type awsSQSConnector struct {
+	sqsURL        string
+	sqsClient     *sqs.SQS
+	connectordata common.ConnectorMetadata
+	logger        *zap.Logger
+}
+
+func (conn awsSQSConnector) consumeMessage() {
+	var maxNumberOfMessages = int64(10) // Process maximum 10 messages concurrently
+	var waitTimeSeconds = int64(5)      //Wait 5 sec to process another message
+
+	headers := http.Header{
+		"KEDA-Topic":          {conn.connectordata.Topic},
+		"KEDA-Response-Topic": {conn.connectordata.ResponseTopic},
+		"KEDA-Error-Topic":    {conn.connectordata.ErrorTopic},
+		"Content-Type":        {conn.connectordata.ContentType},
+		"KEDA-Source-Name":    {conn.connectordata.SourceName},
+	}
+
+	consQueueURL := conn.sqsURL + os.Getenv("TOPIC")
```

Comment:
while joining url use this `path.Join(u.Path, "bar.html")` as this would handle the scenario of '/' present or not.

Reply:
can't directly use `path.join` as the sqs url has account id as well, need to use `ResolveReference` from url package implemented the same
` https://golang.org/pkg/net/url/#example_URL_ResolveReference`